### PR TITLE
Update Go version to 1.20.5

### PR DESCRIPTION
**Pull request by mansinahar**

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
 		"dockerfile": "Dockerfile",
 		"args": {
 			// Update the VARIANT arg to pick a version of Go
-			"VARIANT": "1.19",
+			"VARIANT": "1.20",
```
Comment (mansinahar) [inline .devcontainer/devcontainer.json:9]: I don't use the `devcontainer` setup and so haven't tested this line here. Can somebody who uses this test it please?

File: README.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ Please consider [registering your Prebid Server](https://docs.prebid.org/prebid-
 
 ## Installation
 
-First install [Go](https://golang.org/doc/install) version 1.18 or newer.
+First install [Go](https://golang.org/doc/install) version 1.19 or newer.
```
Comment (VeronikaSolovei9) [inline README.md:24]: 20?

**Review by VeronikaSolovei9 — COMMENTED** [review]
Local testing looks good.

**Comment by AlexBVolcy** [comment]
Local testing looks good, just waiting on the comment Veronika left to be addressed

File: README.md (hunk 2 above)
Comment (mansinahar) [reply README.md:24]: No because we want to mention here the oldest version we support which is 1.19

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (mansinahar) [reply .devcontainer/devcontainer.json:9]: I was able to test the devcontainer and it worked fine with this version update.

**Review by AlexBVolcy — APPROVED** [review]

**Review by VeronikaSolovei9 — APPROVED** [review]

**Pull request merged**